packages/account-v2/src/components/CountrySelector/CountrySelector.tsx

```diff
@@ -23,8 +24,7 @@ export const CountrySelector = ({ disabled, errorMessage, label, name, ...field
             label={label}
             list={residenceList}
             name={name}
-            /*eslint-disable @typescript-eslint/no-empty-function */
-            onSelect={() => {}}
+            onSelect={value => handleSelect?.(value as string)}
```

Comment:
Please add here TODO to clean types later

Reply:
The eslint Rule is removed as the functionality is implemented